app/src/main/java/com/thisisnotajoke/hueyo/controller/HueyoService.java

```diff
@@ -38,8 +41,9 @@
 
 public class HueyoService extends Service {
     private static final String TAG = "HueyoService";
+	private static final int ONGOING_NOTIFICATION_ID = 101;
     private final IBinder mBinder = new LocalBinder();
-
+		
```

Comment:
i just did one instance ill replace tabs with spaces in my settings. i don't see any trailing spaces in my editor.